packages/api-reference/src/features/Operation/helpers/combine-params.test.ts

```diff
@@ -44,18 +44,25 @@ describe('combineParams', () => {
   it('filters out unresolved references', () => {
     const pathParams: any[] = [
       { name: 'id', in: 'path', required: true },
-      { $ref: '#/components/parameters/UnresolvedParam' }, // Unresolved reference
+      {
+        $ref: '#/components/parameters/UnresolvedParam',
+        '$ref-value': { name: 'username', in: 'query', required: false },
+      },
     ]
```

Comment:
we expected an unresolved reference first and now expect a resolved reference?

Reply:
this is an unresolved reference in this context actually